src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -322,6 +316,55 @@ struct ScalarOp<ONNXTanOp> {
   using IOp = NotSuportedScalarOp;
 };
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXIsInfOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXIsInfOp> {
+  using FOp = CustomScalarOp;
+  using IOp = NotSuportedScalarOp;
+};
+
+template <>
+Value emitScalarOpFor<ONNXIsInfOp>(ConversionPatternRewriter &rewriter,
+    Location loc, Operation *op, Type elementType,
+    ArrayRef<Value> scalarOperands) {
+
+  Value operand = scalarOperands[0];
+  Type inputElemType = getElementType(operand.getType());
+  Value result;
+
+  CheckIfCustomScalarOpIsSupported<ONNXIsInfOp>(inputElemType);
+  MultiDialectBuilder<MathBuilder> create(rewriter, loc);
+  Value negInf = create.math.negativeInf(inputElemType);
+  Value posInf = create.math.positiveInf(inputElemType);
+
+  double detectNegAttribute = dyn_cast<ONNXIsInfOp>(op).getDetectNegative();
+  double detectPosAttribute = dyn_cast<ONNXIsInfOp>(op).getDetectPositive();
+
+  // Three different cases: Infinity, Negative Infinity and Positive Infinity
+  bool detectInf = detectPosAttribute == 1 && detectNegAttribute == 1;
+  bool detectNeg = detectNegAttribute == 1 && detectPosAttribute == 0;
+  bool detectPos = detectPosAttribute == 1 && detectNegAttribute == 0;
```

Comment:
Do we have `detectPosAttribute == 0 && detectNegAttribute == 0`? If not, we can verify it inside the op's verifier.

Reply:
Checking it inside IsInf's verifier would be good enough I believe, and you can assert it here if necessary.

Reply:
Okay thank you for your feedback!

Reply:
I added the verifier @tungld

Reply:
Thanks!

BTW, you need to add `IsInf` to the verifier list in the file `utils/gen_onnx_mlir.py` and run `make OMONNXOpsIncTranslation` to update the ONNX definition file, which will solve the issue in Jenkins.

Reply:
Fixed it!